pyaerocom/test/test_colocation.py

```diff
@@ -82,6 +81,27 @@ def test_colocate_gridded_ungridded_nonglobal(aeronetsunv3lev2_subset):
 def test_colocate_gridded_gridded(mod, obs, addargs, **kwargs):
     pass
 
+from pyaerocom.colocation_auto import Colocator
+from pyaerocom.io.read_emep import ReadEMEP
+from pyaerocom.io.readgridded import ReadGridded
+@pytest.mark.parametrize('reader_id,reader_class', [
+    ('ReadEMEP', ReadEMEP),
+    ('ReadGridded', ReadGridded)
+    ])
+def test_colocator_reader(reader_id, reader_class):
+    col = Colocator(gridded_reader_id=reader_id)
+    reader = col.get_gridded_reader()
+    assert reader == reader_class
+
+@testdata_unavail
+def test_colocator_instantiate_model_reader(path_emep):
+    col = Colocator(gridded_reader_id='ReadEMEP')
+    col.filepath = path_emep['daily']
+    r = col.instantiate_model_reader()
+    assert isinstance(r, ReadEMEP)
+    assert r.filepath == col.filepath
+
+
```

Comment:
Can you move these tests into a new test module `test_colocation_auto.py`?  `test_colocation.py` should only test methods / classes in `colocation.py` (pytest convention). This will also be a good first step for #132)

Reply:
Done.